terraform/modules/kubernetes/azure_redis.tf

```diff
@@ -70,9 +82,17 @@ resource "azurerm_redis_cache" "redis-queue" {
   sku_name                      = var.redis_queue_sku_name
   minimum_tls_version           = var.redis_minimum_tls_version
   public_network_access_enabled = var.redis_public_network_access_enabled
+  zones                         = var.redis_replica_zones
 
   redis_configuration {
     maxmemory_policy = "noeviction"
+    #  aof_backup_enabled              = var.enable_aof_backup
+    #  aof_storage_connection_string_0 = var.enable_aof_backup == true ? azurerm_storage_account.storeacc.0.primary_blob_connection_string : null
+    #  aof_storage_connection_string_1 = var.enable_aof_backup == true ? azurerm_storage_account.storeacc.0.secondary_blob_connection_string : null
+    rdb_backup_enabled              = var.redis_backup_enable
+    rdb_backup_frequency            = var.redis_backup_frequency
+    rdb_backup_max_snapshot_count   = var.redis_snapshot_count
+    rdb_storage_connection_string   = azurerm_storage_account.redis_cache_backup[0].primary_blob_connection_string
```

Comment:
Could this be set to either `null` or whatever is nothing or the connection string only if we have specified that we want to enable backup?